bootstrap/scripts/snapshot.sh

```diff
@@ -12,5 +12,5 @@ set -e
 . ${BOOTSTRAP_REPOSITORY:-.}/bootstrap/scripts/envvars.sh
 
 pushd "${BOOTSTRAP_REPOSITORY}"
-tar czf "${BOOTSTRAP_REPOSITORY}/snapshot.tar.gz" Pharo.image Pharo.changes pharo pharo-ui pharo-vm pharo-local vmtarget bootstrap-cache
+tar czf "${BOOTSTRAP_REPOSITORY}/snapshot.tar.gz" Pharo.image Pharo.changes pharo pharo-ui pharo-vm pharo-local ${CACHE}
```

Comment:
it should be BOOTSTRAP_CACHE here

Reply:
Oops, sorry about that.

Actually snapshot.sh and restore.sh were both assuming that bootstrap.sh was being run from within the respository.  Now they should be run under the same conditions as bootstrap.sh.